rp_redcap/tests/test_tasks.py

```diff
@@ -608,6 +608,45 @@ def test_get_choices(self):
         self.assertEqual(roles["0"], "Lead (Investigator, Detective)")
         self.assertEqual(roles["1"], "Investigator")
 
+    def test_save_answers(self):
+        no_update_row = {"my_field": "original_value"}
+        update_row = {"my_field": "new_value"}
+        survey = Survey.objects.create(
+            name="survey_ONE",
+            description="Survey One",
+            rapidpro_flow="f5901b62",
+            urn_field="mobile",
+            project=self.project,
+            sequence=1,
+        )
+        contact = Contact.objects.create(
+            project=self.project, record_id=1, urn="+27123"
+        )
+        answer = SurveyAnswer.objects.create(
+            contact=contact,
+            survey=survey,
+            name="my_field",
+            value="original_value",
+        )
+
+        yesterday = datetime.datetime.now() - datetime.timedelta(days=1)
```

Comment:
We should be using [`timezone`](https://docs.djangoproject.com/en/2.1/topics/i18n/timezones/#naive-and-aware-datetime-objects) for django things, instead of straight `datetime`.